CONTRIBUTING.rst

```diff
@@ -237,16 +237,18 @@ Open Issues
 If you're interested in participating in yt development, take a look at the
 `issue tracker on GitHub
 <https://github.com/yt-project/yt/issues>`_.
-Issues are marked with a milestone of "easy", "moderate", or "difficult"
-depending on the estimated level of difficulty for fixing the issue. While we
-try to triage the issue tracker regularly, it may be the case that issues marked
-"moderate" are actually easier than their milestone label indicates since that
-is the default value.
+You can search by labels, indicating estimated level of difficulty or category,
+to find issues that you would like to contribute to.  Good first issues are
+marked with a label of `new contributor friendly`.  While we try to triage the
+issue tracker regularly to assign appropriate labels to every issue, it may be
+the case that issues not marked as `new contributor friendly` are actually
+suitable for new contributors.
```

Comment:
In the future, we may also want to add a link here for our weekly triage meetings. I don't think that's necessary for this PR to get merged, but it certainly would be a nice inclusive effort for potential new contributors!